docs/source/release_notes/lts/52_0.rst

```diff
@@ -0,0 +1,162 @@
+========
+52.0 LTS
+========
+
+The Avocado team is proud to present another release: Avocado version
+52.0, the second Avocado LTS version.
+
+What's new?
+===========
+
+When compared to the last LTS (v36), the main features introduced by
+this versions are:
+
+- TAP[2] version 12 support.
+- The download of assets in tests now allow for an expiration time.
+- Environment variables can be propagated into tests running on remote
+  systems.
+- The plugin interfaces have been moved into the
+  ``avocado.core.plugin_interfaces`` module.
+- Support for running tests in a Docker container.
+- Introduction of the "Fail Fast" feature (``--failfast on`` option) to
+  the ``run`` command, which interrupts the Job on a first test failure.
+- Special keyword ``latest`` for replaying previous jobs.
+- Support to replay a Job by path (in addition to the Job ID method and
+  the ``latest`` keyword).
+- The introduction of a tool that generated a diff-like report of two
+  jobs.
+- The introduction of a ``rr`` based wrapper.
+- The automatic VM IP detection that kicks in when one uses
+  ``--vm-domain`` without a matching ``--vm-hostname``, now uses a more
+  reliable method (libvirt/qemu-gust-agent query).
+- Set ``LC_ALL=C`` by default on sysinfo collection to simplify avocado
+  diff comparison between different machines.
+- Result plugins system is now pluggable and the results plugins (JSON,
+  XUnit, HTML) were turned into stevedore plugins. They are now listed
+  in the ``avocado plugins`` command.
+- Multiplexer was replaced with Varianter plugging system with defined
+  API to register plugins that generate test variants.
+- Old ``--multiplex`` argument, which used to turn yaml files into
+  variants, is now handled by an optional plugin called ``yaml_to_mux``
+  and the ``--multiplex`` option is being deprecated in favour of
+  the ``--mux-yaml`` option, which behaves the same way.
```

Comment:
actually this is very nice for the "what changed" section. For new-features I'd just say: A new variant-generating interface was developed called `Varianter` and it allows concurrent use of multiple plugins providing test variants.